### PR TITLE
Add property binding for testFilter CLI option [ATLAS-850]

### PR DESCRIPTION
**Pull request by Azurelol**
## Description
Added property bindings (env/properties) for the unity command line option `testFilter`

## Changes
* ![ADD] `UnityPluginConvention` Added `testFilter` property lookup

[NEW]:      https://resources.atlas.wooga.com/icons/icon_new.svg "New"
[ADD]:      https://resources.atlas.wooga.com/icons/icon_add.svg "Add"
[IMPROVE]:  https://resources.atlas.wooga.com/icons/icon_improve.svg "Improve"
[CHANGE]:   https://resources.atlas.wooga.com/icons/icon_change.svg "Change"
[FIX]:      https://resources.atlas.wooga.com/icons/icon_fix.svg "Fix"
[UPDATE]:   https://resources.atlas.wooga.com/icons/icon_update.svg "Update"

[BREAK]:    https://resources.atlas.wooga.com/icons/icon_break.svg "Remove"
[REMOVE]:   https://resources.atlas.wooga.com/icons/icon_remove.svg "Remove"
[IOS]:      https://resources.atlas.wooga.com/icons/icon_iOS.svg "iOS"
[ANDROID]:  https://resources.atlas.wooga.com/icons/icon_android.svg "Android"
[WEBGL]:    https://resources.atlas.wooga.com/icons/icon_webGL.svg "WebGL"
[GRADLE]:   https://resources.atlas.wooga.com/icons/icon_gradle.svg "GRADLE"
[UNITY]:    https://resources.atlas.wooga.com/icons/icon_unity.svg "Unity"
[LINUX]:    https://resources.atlas.wooga.com/icons/icon_linux.svg "Linux"
[WIN]:      https://resources.atlas.wooga.com/icons/icon_windows.svg "Windows"
[MACOS]:    https://resources.atlas.wooga.com/icons/icon_iOS.svg "macOS"

### CODE REVIEW

File: src/main/groovy/wooga/gradle/unity/UnityPlugin.groovy [hunk 1]
```diff
@@ -148,6 +151,7 @@ class UnityPlugin implements Plugin<Project> {
             t.logToStdout.convention(t.logger.infoEnabled || t.logger.debugEnabled)
             t.toggleLogFile(true)
             t.buildTarget.convention(extension.defaultBuildTarget)
+            t.testFilter.convention(extension.testFilter)
```
Comment (Larusso) [inline src/main/groovy/wooga/gradle/unity/UnityPlugin.groovy:154]: I think this should only be added to the `Test` tasks. See line 200

**Review by Larusso — CHANGES_REQUESTED** [review]

File: src/main/groovy/wooga/gradle/unity/UnityPlugin.groovy (hunk 1 above)
Comment (Azurelol) [reply src/main/groovy/wooga/gradle/unity/UnityPlugin.groovy:154]: Good point

**Review by Larusso — APPROVED** [review]

**Comment by minitrope** [comment]
Is there anything else needed to merge this PR? I need it to merge one of my own.

**Comment by Azurelol** [comment]
> Is there anything else needed to merge this PR? I need it to merge one of my own.

There's not. I apologize, having missed its approval.

**Comment by minitrope** [comment]
No worries, thanks for taking care of it 👍

**Pull request merged**